### PR TITLE
Bump avago rc

### PR DESCRIPTION
**Pull request by ceyonur**
bumps to avalancheGo RC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,44 +18,6 @@ The Subnet EVM runs in a separate process from the main AvalancheGo process and
 ### AvalancheGo Compatibility
 
 ```text
-[v0.1.0] AvalancheGo@v1.7.0-v1.7.4 (Protocol Version: 9)
```
Comment (StephenButtolph) [inline README.md:21]: Why was this removed? (Or why was it ever here?)

Comment (ceyonur) [reply README.md:21]: I think this was a request from CLI/AvaCloud team. I don't think we need those old versions. We can even consider removing this altogether, and only keep the current supported versions in readme or in release notes. @ava-labs/platform-evm

**Review by ARR4N — APPROVED** [review]
